### DISCUSSION TITLE
XAML QUESTION

### DISCUSSION THREAD

**Discussion by Tana64** [WR-TAFESA/Tafe240262#2] [Ideas]
Quick question Wesley. I'm a tad rusty in XAML. How can I set the background color for a TextBox in XAML ?

**Comment by WR-TAFESA** [comment]
You can use Background="" to change a text box's fill colour. XAML accepts a bunch of different colour names, or you can use a #FFFFFF hex code to get an exact colour.

**Reply by Tana64** [reply]
Thanks man ! Much appreciated
